src/partials/products-section.html

```diff
@@ -1 +1,32 @@
-<h1>Hello team</h1>
\ No newline at end of file
+<section class="products">
+  <div class="container">
+    <p class="products__label">100% natural</p>
+    <h2 class="products__title">PRODUCTS</h2>
+    <ul class="products__list">
+      <li class="products__item">
```

Comment:
products__list-item

Reply:
mam gandit si eu ca aici nu prea e ok, chiar ma gandeam sa ma mai uit putin peste BEM

Reply:
in partea asta doar ce ti-am zis, in rest mi se pare ok structura